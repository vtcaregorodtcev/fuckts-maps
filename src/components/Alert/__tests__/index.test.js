import React from 'react';
// eslint-disable-next-line no-unused-vars
import { h } from 'preact';
import renderer from 'react-test-renderer';
import { AlertComponent } from '../index';

it('renders correctly with msg', () => {
  const tree = renderer.create(<AlertComponent msg="test" />).toJSON();
  expect(tree).toMatchSnapshot();
});

it('renders correctly without msg', () => {
  const tree = renderer.create(<AlertComponent />).toJSON();
  expect(tree).toMatchSnapshot();
});
