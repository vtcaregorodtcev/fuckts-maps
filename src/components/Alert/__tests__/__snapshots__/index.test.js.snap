// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly with msg 1`] = `
<div>
  <div
    className="ant-alert ant-alert-success"
    data-show={true}
  >
    <i
      aria-label="icon: check-circle"
      className="anticon anticon-check-circle ant-alert-icon"
    >
      <svg
        aria-hidden="true"
        className=""
        data-icon="check-circle"
        fill="currentColor"
        height="1em"
        viewBox="64 64 896 896"
        width="1em"
      >
        <path
          d="M512 64C264.6 64 64 264.6 64 512s200.6 448 448 448 448-200.6 448-448S759.4 64 512 64zm193.5 301.7l-210.6 292a31.8 31.8 0 0 1-51.7 0L318.5 484.9c-3.8-5.3 0-12.7 6.5-12.7h46.9c10.2 0 19.9 4.9 25.9 13.3l71.2 98.8 157.2-218c6-8.3 15.6-13.3 25.9-13.3H699c6.5 0 10.3 7.4 6.5 12.7z"
        />
      </svg>
    </i>
    <span
      className="ant-alert-message"
    >
      test
    </span>
    <span
      className="ant-alert-description"
    />
  </div>
</div>
`;

exports[`renders correctly without msg 1`] = `null`;
